hack/verify-copyright-header.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+year=$(date +'%Y')
+
+GIT_BRANCH=$(git branch | grep \* | cut -d ' ' -f2)
+newly_added_files=($(git diff origin/master ${GIT_BRANCH} --name-only --diff-filter=A))
```

Comment:
We should probably include only certain directories like `pkg/`, `cmd`, etc. Or exclude vendor folder and others where we have no control on.

Reply:
Good point. excluding `vendor` dirs maybe easier.

Reply:
we'll also want to exclude `BUILD` and `BUILD.bazel` file.

Reply:
we might be able to reuse this https://github.com/kubernetes/test-infra/blob/master/hack/verify_boilerplate.py

Reply:
not using this script anymore